dcpy/lifecycle/builds/plan.py

```diff
@@ -341,30 +341,32 @@ def _cli_wrapper_repeat_recipe(
     ),
 ):
     product_key: publishing.BuildKey | publishing.DraftKey | publishing.PublishKey
-
+    product_label = (
```

Comment:
Thoughts on just passing along a `--product-label` CLI option?

Reply:
Something like that would make sense, though then we'd need to have this sort of logic in the `repeat` gha - still need to refer to both formattings in our code at the moment

Reply:
Cool. Yeah I feel like a CLI option isn't really great either, in that it still adds some unneeded complexity. But it's 100% a lesser evil than what's in this PR

Maybe we should just have some sort of minimal `ProductPaths` object? And then dcpy can rely on pyproject.toml (or `.dcpy` file, etc) that defines the path to product folder, DO folder name, db build name, etc. OR for each product, we have a standard spot to look, (i.e. the product folder) and store the additional metadata there. This seems like this could belong in a recipe actually. Though then you would need the recipe to run an upload step. 

Long way of saying I think we should punt the issue. I can make an issue for this